Geometry/TrackerNumberingBuilder/bin/stubs/GeometricDetLoader.h

```diff
@@ -24,6 +24,7 @@ class GeometricDetLoader : public edm::one::EDAnalyzer<edm::one::WatchRuns> {
 
 private:
   void putOne(const GeometricDet* gd, PGeometricDet* pgd, int lev);
+  bool fromDD4hep;
```

Comment:
please, follow the CMS naming conventions

Reply:
I think all three files in the `Geometry/TrackerNumberingBuilder/bin/stubs` directory can be deleted. I don't see any usage of the `GeometricDetLoader` in any C++ or Python file.
@adewit Were you able to test your changes to this file?

Reply:
I didn't run this module specifically, no. Do you want me to remove them in this PR or is it better to do this in a separate PR to give potential users with a private config relying on this code a chance to object?

Reply:
@adewit Two points: 
You can delete `GeometricDetLoader` in a a separate PR if you wish, or go ahead with the deletion in this one. I wouldn't worry too much about it. Anyone who is using it can fetch it out of Git history and continue using it. I probably wouldn't allow it back into CMSSW because I don't want to maintain someone's private code.

Second point: In general, if a file is changed, it needs to be tested. We can't allow changes in CMSSW that have never been tested. If it is not clear how a file gets executed, that needs to be tracked down. If it is discovered that nothing uses a particular file, that raises the important question of whether that file belongs in CMSSW at all.

Reply:
Ok - point 2 decides point 1 in that case - have removed the files from here.